solr/solr-ref-guide/src/the-query-elevation-component.adoc

```diff
@@ -61,7 +61,18 @@ Optionally, in the Query Elevation Component configuration you can also specify
 The Query Elevation Search Component takes the following parameters:
 
 `queryFieldType`::
-Specifies which fieldType should be used to analyze the incoming text. For example, it may be appropriate to use a fieldType with a LowerCaseFilter.
+Specifies which fieldType should be used to analyze the incoming text. For example, it may be appropriate to use a fieldType with a LowerCaseFilter. Other example, if you need to unescape backslash-escaped queries, then you can define the fieldType to preprocess with a PatternReplaceCharFilter. Here is the corresponding example of fieldType (traditionally in `schema.xml`):
+
+[source,xml]
+----
+<fieldType name="unescapelowercase" class="solr.TextField">
+  <analyzer>
+    <charFilter class="solr.PatternReplaceCharFilterFactory" pattern="\\(.)" replacement="$1"/>
```

Comment:
I think there's a syntax to use such that we only un-escape non-alphanumeric?  If they type a\b (which might be in some identifier) then they probably meant that.

Reply:
Indeed, but do we want to go into this level of detail?
The Lucene StandardQueryParser uses EscapeQuerySyntaxImpl.discardEscapeChar() which unescape everything without distinction except it detects unicode \uxxxx and converts it.
I suggest that we give a basic example of unescaping every backslash. If the user wants more sophisticated logic, then she will design the right pattern or the custom CharFilter.

Reply:
The regex would be "\\([^\p{Alnum}])", but it applies to ASCII alphanum, not internationalized.

Reply:
Actually it is possible to localize the pattern with
"\\\\([^\\p{javaLowerCase}\\p{javaUpperCase}\\p{Digit}])"

Reply:
I added this other pattern as an example.